Pod/Classes/GimbalAdapter.swift

```diff
@@ -1,13 +1,20 @@
-/* Copyright 2017 Urban Airship and Contributors */
+/* Copyright Airship and Contributors */
 
-import AirshipKit
+
+#if canImport(AirshipCore)
```

Comment:
Should just be Airship right? Since its a pod we should expect that they are using pods?